src/ceph-volume/ceph_volume/tests/functional/lvm/centos7/bluestore/create/setup.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
If this playbook is the same in all the scenarios that uses it can we store it in a central location and use symlinks?

Reply:
this has been addressed